src/applications/caregivers/components/SubmitError/DownloadLink.jsx

```diff
@@ -24,15 +31,24 @@ const DownLoadLink = ({ form }) => {
           },
         },
       )
-        .then(response => {
-          return response.blob();
-        })
+        .then(response => response.blob())
         .then(blob => {
           const url = URL.createObjectURL(blob);
           setPDFLink(url);
+          recordEvent({ event: 'caregivers-10-10cg-pdf-download-success' });
+        })
+        .catch(error => {
+          Sentry.withScope(scope => {
```

Comment:
I added sentry since we are catching it, Sentry would no longer log it. We wanted to track it in GA so we can track PDF interactions. This way we can have best of both worlds. I am not logging any data so PII should be safe.